homeassistant/components/renault/sensor.py

```diff
@@ -333,4 +334,19 @@ def _get_utc_value(entity: RenaultSensor[T]) -> datetime:
         name="Location Last Activity",
         value_lambda=_get_utc_value,
     ),
+    RenaultSensorEntityDescription(
+        key="res_state",
+        coordinator="res_state",
+        data_key="details",
+        entity_class=RenaultSensor[KamereonVehicleResStateData],
+        name="Remote Engine Start State",
```

Comment:
You can drop `State`, it's an entity in the state machine after all. 
```suggestion
        name="Remote Engine Start",
```